src/jit/compiler.h

```diff
@@ -7293,14 +7293,13 @@ class Compiler
                            unsigned refCntWtdStkDbl);
 #endif // DOUBLE_ALIGN
 
-    __declspec(property(get = getFullPtrRegMap, put = setFullPtrRegMap)) bool genFullPtrRegMap;
-    bool getFullPtrRegMap()
+    bool GetFullPtrRegMap()
     {
-        return codeGen->genFullPtrRegMap;
+        return codeGen->IsFullPtrRegMapRequired();
     }
-    void setFullPtrRegMap(bool value)
+    void SetFullPtrRegMap(bool value)
     {
-        codeGen->setFullPtrRegMap(value);
+        codeGen->SetFullPtrRegMap(value);
```

Comment:
This now seems rather inconsistent. I'm not sure if we have an existing example of this, but:
- I think that the `CodeGen` and `Compiler` versions should be named the same (or just keep the one on `CodeGen`)
- I think the `Set` method should correspond, e.g. `SetFullPtrRegMapRequired`